pkg/services/ngalert/migration/cond_trans.go

```diff
@@ -265,7 +307,7 @@ func getNewRefID(refIDs map[string][]int) (string, error) {
 		}
 		return sR, nil
 	}
-	return "", fmt.Errorf("failed to generate unique RefID")
+	return "", fmt.Errorf("generate unique RefID")
```

Comment:
I do not think I can agree with this. This makes the chained message be more readable message `failed to migrate alert rules: failed to migrate alert rule:failed to generate unique RefID` is more readable than `migrate alert rules: migrate rule: generate unique RefID`.

Reply:
Yes, you're right. The leaf errors should mention the failure, this was accidental collateral in a mass replace.

Reply:
Though in this case it would be:

`Error "migration failed: executing migration: migrate org 1: migrate alerts: migrate and save dashboard '1': migrate alert 'alert 1': transform conditions: failed to generate unique RefID"`